lib/pinyin.ex

```diff
@@ -458,7 +458,13 @@ defmodule Pinyin do
     final = String.replace(f, "v", "ü")
     vowel = final |> String.codepoints() |> Enum.reduce(nil, &select_max/2)
 
-    i <> String.replace(final, vowel, Pinyin.Char.with_tone(vowel, t))
+    # TODO: Enable this for non-vowels
```

Comment:
That means adding it to `select_max/2`

Reply:
> That means adding it to `select_max/2`

Yeah that doesn't work...

Need a special case here.

Reply:
I've adjusted `marked` to handle this.